kytos/templates/kytos.conf.template

```diff
@@ -78,15 +79,41 @@ vlan_pool = {}
 # The jwt_secret parameter is responsible for signing JSON Web Tokens.
 jwt_secret = {{ jwt_secret }}
 
+# Time to expire authentication token in minutes
+token_expiration_minutes = 180
+
 # Maximum number of thread workers in a thread pool, if it's set as empty dict {} no thread pools will be used.
 # The following pools are available by default to be used in the listen_to decorator:
 # - sb: it's used automatically by kytos/of_core.* events, it's meant for southbound related messages
 # - app: it's meant for general NApps event, it's default pool if no other one has been specified
 # - db: it can be used by for higher priority db related tasks (need to be parametrized on decorator)
 thread_pool_max_workers = {"sb": 256, "db": 256, "app": 512}
 
-# Time to expire authentication token in minutes
-token_expiration_minutes = 180
+# Configuration for KytosEventBuffers
+# Valid event buffers are "msg_in", "msg_out", "app", "conn", and "raw".
+# Valid buffer types are:
+#  - "default": Use KytosEventBuffer. Has one optional parameter "queue", which determines the type of queue to use, either "default" for a standard queue or "priority" for a priority queue.
+# - "rate_limited": Use RateLimitedBuffer. This extends "default", and inherits the options from that. It has two options "strategy" and "limit". "strategy" determines the "type" of rate limiting strategy to use and what "storage" to use for the rate limiting data. Valid "type"s of strategy are "moving_window", "fixed_window", and "elasitc_window". Only one "type" of storage is supported "memory". "limit" sets the limit on how fast messages may be processed after they have been received by the event buffer.
+
+event_buffer_conf =
+    {
+        "msg_out": {
+            "type": "rate_limited",
+            "queue": "priority",
+            "strategy": {
+                "type": "moving_window",
+                "storage": {
+                    "type": "memory"
+                }
+            },
+            "limit": "100/second",
+            "identifier": "destination.id"
```

Comment:
The current config format and spec cover the main case we have in mind. It might be worth assessing if the identifier should actually be a logical and of attrs, just so it can get combined to rate limit let's say `destination.id` and `message.header.message_type.value`. And, it might be worth also assessing if we should support out of the gate multiple chained rate limiters (with their strategy) with their identifiers per queue. We certainly don't want to allow way too many per queue (and there's a lof of possibilities here), but maybe supporting up to 5 or so in the first iteration? It might end up with an API that would also be a bit more flexible. I'll leave this point for you to think about it (and it won't be a blocker for this current PR), maybe map a new issue for the discussion, and then we can officialize if we'll only support a single one or multiple ones before bumping version `2023.2`

Reply:
I think what we should potentially look into the future is sorting KytosEvents into separate pools. Currently if the rate limit is reached for any given identifier, then the current thread/coroutine has to wait until that rate limit is cleared, before it can process a request for any other given identifier. The current system works well enough to stop sudden bursts of events, but I don't think its extensible to chained rate limiters unless you do per connection pooling first.

However, I can support multiple attrs, and in fact zero attrs as well for the identifiers per a rate limiter, so I'll implement at least that.

Reply:
Right @Ktmi, let's keep this in mind and reassess in the future. Yes, the current implementation with a single limiter and multiple attributes is already solid enough for the typical use case as far as we currently know that will be used in prod.